c2cgeoportal/scaffolds/create/vars_+package+.yaml_tmpl

```diff
@@ -115,6 +115,10 @@ vars:
             The GeoMapfish team
         smtp_server: smtp.example.com
 
+    headers:
+        mapserver:
+            cors: *
```

Comment:
Do be able to use it in the `WMSBrowser` of an other site without using the `OGCproxy` who is also a potential hole.